spec/e2e/migration_spec.rb

```diff
@@ -13,7 +13,7 @@ class ThirdRelClass; end
       class User
         include Neo4j::ActiveNode
         property :name
-        has_many :out, :songs, model_class: :Song, type: 'songs'
+        has_many :out, :songs, model_class: :'MigrationSpecs::Song', type: 'songs'
```

Comment:
Ah, nice refactor... rebased this branch off it, will handle the docs tomorrow :smile: